### PR TITLE
Add grafana dashboards and configmap creation for auto provisioning

### PR DESCRIPTION
**Pull request by kanuahs**
## Which issue(s) this PR fixes:

Fixes #

## Checklist:

- [x] I ran tests as well as code linting locally to verify my changes. 
- [x] I have done manual verification of my changes, changes working as expected.
- [ ] I have added new tests to cover my changes.
- [x] My changes follow contributing guidelines of Fission.
- [ ] I have signed all of my commits.

### CODE REVIEW

**Comment by kanuahs** [comment]
image
Archive metrics panels in fission admin dashboard

**Comment by sanketsudake** [comment]
@kanuahs please rebase PR with latest main

File: charts/fission-all/templates/dashboards/dashboards.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{{- $files := .Files }}
+{{- range $path, $bytes := .Files.Glob "dashboards/*.json" }}
+{{- $filename := trimSuffix (ext $path) (base $path) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-%s" $.Chart.Name $filename | trunc 63 | trimSuffix "-" }}
+  namespace: {{ $.grafana.Namespace }}
+  labels:
+    grafana_dashboard: "1"
```
Comment (sanketsudake) [inline charts/fission-all/templates/dashboards/dashboards.yaml:11]: Should we add fission reference here ? any corrections needed here ?

Comment (kanuahs) [reply charts/fission-all/templates/dashboards/dashboards.yaml:11]: Thats the default label used by the grafana sidecar in kube prometheus stack to query for dashboards. I can change it to something like this https://github.com/prometheus-community/helm-charts/blob/cb31a4d4ca79ea92aa0628050111efeada5f7df9/charts/kube-prometheus-stack/templates/grafana/configmap-dashboards.yaml#L15 and add another label like `created-by:fission`

Comment (sanketsudake) [reply charts/fission-all/templates/dashboards/dashboards.yaml:11]: yes this also looks also good. 
Additionally we can add `created-by: fission`

**Pull request merged**